### DISCUSSION TITLE
zh/guide/install/paas

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#313] [Announcements]
# zh/guide/install/paas

具体用法请参考对应仓库中的README.md。 Koyeb https://github.com/alist-org/alist-koyeb Render https://github.com/alist-org/alist-render Heroku https://github.com/alist-org/alist-heroku-postgres...

https://alist.nn.ci/zh/guide/install/paas.html

**Comment by ilxp** [comment]
Sealos 部署后，默认登陆密码是多少呢？

**Reply by ilxp** [reply]
找了半天，才找到终端。感谢。应用管理（日志，终端，详情，重启） 然后输入：./alist admin set NEW_PASSWORD

**Comment by ningfeiyu** [comment]
部署之后web后端前端都是正常的，但是要用infuse和nplayer连不上服务器。。

**Reply by ningfeiyu** [reply]
端口换成443就可以了。

**Comment by eliguoguo** [comment]
请问用 Koyeb 部署，总是提示 Instance is unhealthy ，最后 Instance has exited with code 1 ，换了不同的数据库都不行，到底是哪里出了问题，该怎么解决？

**Reply by Wefbt** [reply]
111，同样的问题，是因为现在的免费计划配置不够用吗，你用的是什么配置的啊
